utils/userData.ts

```diff
@@ -49,6 +50,7 @@ export const AllUserData = t.interface({
     "NotificationStatusList"
   ),
   notifications: t.readonlyArray(t.exact(SafeNotification), "NotificationList"),
-  profiles: t.readonlyArray(t.exact(Profile))
+  profiles: t.readonlyArray(t.exact(Profile)),
+  servicesPreferences: t.readonlyArray(ServicePreference)
```

Comment:
here i couldn't find a way to add a `t.exact` to `ServicePreference` because `t.exact` doesn't support `UnionType`

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/191/commits/e0c86fcd18ace0bebb119961e33c04baa3b756f5